### DISCUSSION TITLE
Kafka-UI AdminClient access

### DISCUSSION THREAD

**Discussion by metalshanked** [provectus/kafka-ui#3032] [Q&A] [answered]
I am trying to connect to a SSL enabled Confluent Kafka cluster which has been ACL'd via the SSL cert.

However, i get "Cannot create admin client" in the logs.  With the super user cert, it works great.

Is there a way to make Kafka UI work with ACL'd SSL certs?  if so, what are exact permissions required?

**Comment by Haarolean** [comment, marked as answer]
Hi, I'd need your exception stacktrace in the first place.
